### PR TITLE
Educational content: add some exercises about short strings

### LINKED ISSUE #132: Cairo short strings basic exercise
## Description
Implement a basic exercise to understand how short strings are defined in Cairo

## Acceptance criteria
 - The exercise shows how Cairo handles felt-encoded strings
 - The contract compiles and tests pass
 - An exercise patch is generated

### PR DESCRIPTION
**Pull request by tekkac**
Closes #132

### CODE REVIEW

File: exercises/strings/strings00.cairo [hunk 1]
```diff
@@ -0,0 +1,25 @@
+%lang starknet
+
+# Cairo supports short strings which are encoded as ASCII under the hood
+# e.g. let hello_string = 'Hello'
+#      let hello_string = 0x48656c6c6f
```
Comment (ofux) [inline exercises/strings/strings00.cairo:5]: did you mean `hello_hex`?

File: exercises/strings/strings00.cairo [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#      let hello_string = 310939249775
```
Comment (ofux) [inline exercises/strings/strings00.cairo:6]: did you mean `hello_felt`?

File: exercises/strings/strings00.cairo [hunk 3]
```diff
@@ -0,0 +1,25 @@
+%lang starknet
+
+# Cairo supports short strings which are encoded as ASCII under the hood
```
Comment (ofux) [inline exercises/strings/strings00.cairo:3]: I think adding a link to a corresponding doc could be useful, if it exists :)

**Review by ofux — COMMENTED** [review]
Excellent exercise that I'm actually going to do to learn more about strings!
Just a few minor comments

File: exercises/strings/strings00.cairo (hunk 3 above)
Comment (Bernardstanislas) [reply exercises/strings/strings00.cairo:3]: Like for some other sections you can add a `README.md` file in the `strings` folder to point out to some doc or give more in-depth explanations about strings.

**Review by Bernardstanislas — APPROVED** [review]

**Pull request merged**